easybuild/easyconfigs/k/kalign2/kalign2-3.3.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'kalign2'
+version = '3.3.1'
+
+homepage = ''
```

Comment:
@sassy-crick as above, https://msa.sbc.su.se/cgi-bin/msa.cgi looks more or less to be the homepage.

Reply:
Here I am not so sure. The author wrote to me:
" The https://msa.sbc.su.se/ site is not maintained by myself. My recommendation would be to go with
the GitHub which corresponds to the recently improved version 3 of the software."
I have thus added the GitHub page for this EC. 
That was the thing I was pondering about and when it all build, I forgot to insert the home page. 
What is your adivce?